script/Ciel/Day 8/08_01_CIEL08_5.txt

```diff
@@ -835,13 +842,13 @@ I have maintained regular contact, and I'm allowed to act independently, aren't
 -- Page 476, Offset 19458.
 -- 「代行者シエルはテメェとのコンビを解くとよ。できうるかぎり早くテメェを引き取れなんて言ってきやがった。
 "Executor Ciel's cutting you loose.
- She told me to come pick your ass up as soon as I could.
+ Told me to come pick your ass up as soon as I could.
 }
 [sha:62cb8dc79b182a4c9494ae6a0870d05f9a4fcdb6]{
 -- Page 476, Offset 19459.
 -- なんで、近いうちにテメェはオレの預かりだが……スコアも悪い、精神も不安定な不良物件を使うほどマヌケじゃねえ。
 // Again the 'score' thing. Not really worth calling back imo
-So, them's the breaks. You're my problem now, but... your performance sucks, and you've got more than a few screws loose.
+So, them's the breaks. You're %{i}my%{/i} problem now, but... your performance sucks, and you've got more than a few screws loose.
  I'm not dumb enough to use a gun that might go off in my face.
 }
 [sha:12b71e01ce2940729d4f0611369fa38aed159ef2]{
```

Comment:
Below, at least this first mention by Mario of the monastery should be "a monastery"
Offset 19461 is fine though.